test/sa2ref_test.cpp

```
@@ -1,135 +1,135 @@
+    while(numSmems < MAX_NUM_SMEM) {
+		int32_t val = 0;
+        SMEM smem;
+        smem.rid = 0;
+        smem.m = 0;
+        smem.n = 0;
+        smem.l = 0;
+        smem.k = 0;
+        smem.s = 0;
+		if ((val = fscanf(infp, "%ld, %ld", &(smem.k), &(smem.s))) == -1)
+			break;
+		smemArray[numSmems++] = smem;
```

Cppcheck (2.20.0)

[error] Uninitialized variable

(uninitvar)